### ISSUE TITLE
custom permalink structure not recognized

### ISSUE THREAD

**Issue by phoshenwen** [ashhitch/gatsby-plugin-wpgraphql-seo#2]
Maybe not so much an issue as it is a discussion.

I changed WordPress' permalink structure to be `/blog/%postname%`, but when pulling the SEO data via graphql I don't see `/blog/` reflected anywhere in there. As I am using this plugin for breadcrumbs as well I am technically missing a step in between "Home" and the blogpost title.

I've already downloaded the Yoast Test Helper plugin to clear its internal cache but that didn't help (helped to recognize permalink structure in the past, unrelated to Gatsby & this plugin).

In summary, the url pulled by the plugin correctly reflects the permalink structure, but I am only getting 2 objects in the array, instead of 3. 

```
  "data": {
    "wpPost": {
      "seo": {
        "breadcrumbs": [
          {
            "text": "Home",
            "url": "/"
          },
          {
            "text": "Hello world!",
            "url": "/blog/hello-world/"
          }
        ]
      }
    }
  },
```

Is there any way to include "blog" in between Home and the blog post?

**Comment by DanielRiveraHQ** [comment]
@phoshenwen did you solved this one?

**Comment by phoshenwen** [comment]
@DanielRiveraHQ No I have no fix for this currently.

**Comment by snibbo71** [comment]
I've actually only just seen this, but I've opened a related issue to do with breadcrumbs as that url is invalid in any case - it needs to be the full URL or Google Search Console will start emailing about badly formed breadcrumbs.
